components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Cdm/src/Services/CdmMessageProcessor.cs

```diff
@@ -301,209 +356,198 @@ public class CdmMessageProcessor : ICdmClient {
+            persist = false;
+            if (string.IsNullOrEmpty(key) || sample == null) {
+                return null;
             }
-            var csvTrait = partition.ExhibitsTraits.Item("is.partition.format.CSV");
-            partitionLocation = _cdmCorpus.Storage.CorpusPathToAdapterPath(partition.Location);
-            partitionDelimitor = csvTrait?.Arguments?.FetchValue("delimiter") ?? kCsvPartitionsDelimiter;
-            return true;
-        }
 
-        private bool CreateEntityData(DataSetMessageModel dataSet,
-            out string partitionLocation, out string partitionDelimitor) {
-            var key = GetNormalizedEntityName(dataSet.PublisherId,
-                dataSet.DataSetWriterId, dataSet.MetaDataVersion);
             // check if the enetity was aleready added
-            var entity = Manifest.Entities.Item(key);
-            if (entity != null) {
-                return GetEntityData(dataSet, out partitionLocation, out partitionDelimitor);
-            }
+            var entityDefinition = Manifest.Entities.Item(key);
+            if (entityDefinition == null) {
+                // add a new entity for the sample
 
-            // add a new entity for the Message
-            var newSampleEntity = _cdmCorpus.MakeObject<CdmEntityDefinition>(
-                CdmObjectType.EntityDef, key, false);
+                // add a new entity for the Message
+                var newSampleEntity = _cdmCorpus.MakeObject<CdmEntityDefinition>(
+                    CdmObjectType.EntityDef, key, false);
 
-            var info = typeof(DataSetMessageModel).GetProperties();
-            foreach (var property in info) {
-                if (property.Name != nameof(DataSetMessageModel.Payload)) {
+                var info = typeof(MonitoredItemSampleModel).GetProperties();
+                foreach (var property in info) {
                     // add the attributes required
                     var attribute = _cdmCorpus.MakeObject<CdmTypeAttributeDefinition>(
-                    CdmObjectType.TypeAttributeDef, property.Name, false);
+                        CdmObjectType.TypeAttributeDef, property.Name, false);
                     attribute.Purpose = _cdmCorpus.MakeRef<CdmPurposeReference>(
                         CdmObjectType.PurposeRef, "hasA", true);
                     //  if we handle a value, lookup it's type property
+                    /*
                     if (property.Name == "Value" &&
-                        typeof(DataSetMessageModel).
-                            GetProperty("TypeId")?.GetValue(dataSet) is Type typeId) {
+                        typeof(MonitoredItemSampleModel).
+                            GetProperty("TypeId")?.GetValue(sample) is Type typeId) {
                         attribute.DataFormat = DataTypeToCdmDataFormat(typeId);
                     }
                     else {
                         attribute.DataFormat = DataTypeToCdmDataFormat(property.PropertyType);
                     }
```

Comment:
forgotten old code?

Reply:
kind-of yes, this code is no longer used, I'll fix it though